tests/feeds.py

```diff
@@ -20,47 +21,52 @@ def setUpClass(cls) -> None:
         database_arango.db.connect(database="yeti_test")
         database_arango.db.clear()
 
-    def test_feodo_tracker_ip_blocklist(self):
-        defaults = feodo_tracker_ip_blocklist.FeodoTrackerIPBlockList._defaults.copy()
-        defaults["name"] = "FeodoTrackerIPBlocklist"
-        feed = feodo_tracker_ip_blocklist.FeodoTrackerIPBlockList(**defaults)
-        feed.run()
+    # def test_feodo_tracker_ip_blocklist(self):
```

Comment:
Need to uncomment all these, was for debug purposes

Reply:
Whenever you feel ready :)